packages/theme-default/src/layout/DocLayout/docComponents/code/index.tsx

```diff
@@ -51,7 +51,17 @@ export function Code(props: CodeProps) {
         );
       case 'shiki':
       default:
-        return <code {...props}></code>;
+        return (
+          <pre
+            {...props}
+            style={{
+              whiteSpace: codeWrap ? 'pre-wrap' : 'pre',
+              overflowX: 'auto',
```

Comment:
Without this style, too long lines will overflow the whole container, and the code button group will not work as expected:

<img width="1014" alt="image" src="https://github.com/user-attachments/assets/062eb3a2-fdb4-4acd-b5ae-1a7e1804dd53" />